olcPixelGameEngine.h

```diff
@@ -2332,6 +2336,99 @@ namespace olc
 		}
 	}
 
+
+  void PixelGameEngine::DrawPolygon(const std::vector<olc::vi2d>& points, olc::Pixel p)
+  {
+    // Maybe nothing to do?
+    if (points.size() <= 0) 
+      return;
+
+    // Draw every line and close the polygon
+    for (int i=0; i < points.size(); i++) {
+      DrawLine(points[i], points[(i+1) % points.size()], p);
+    }

```

Comment:
```c++
    for (int i=0; i < points.size() - 1; i++) {
      DrawLine(points[i], points[i+1], p);
    }
    DrawLine(*(points.end() - 1), points[0], p);
```
will probably be faster than a modulo on every loop that only occurs once

Reply:
only one way to find out :) but sure, good catch